packages/recommend-react/src/__tests__/useFrequentlyBoughtTogether.test.ts

```diff
@@ -24,22 +27,100 @@ function createMockedRecommendClient() {
 }
 
 describe('useFrequentlyBoughtTogether', () => {
-  test('gets Frequently Bought Together products', () => {
+  test('returns FBT recommendations', async () => {
     const { recommendClient } = createMockedRecommendClient();
 
-    renderHook(() => {
-      const { recommendations } = useFrequentlyBoughtTogether({
+    const { result } = renderHook(() =>
+      useFrequentlyBoughtTogether({
         indexName: 'test',
         recommendClient,
         threshold: 0,
         objectIDs: ['testing'],
         queryParameters: {
           facetFilters: ['test'],
         },
-        transformItems: (items) => items,
-      });
+      })
+    );
+    await waitFor(() => {
+      expect(result.current.recommendations).toEqual([hit]);
+    });
+  });
+
+  test('assures that the transformItems function always returns an array', async () => {
```

Comment:
We can remove this test now, it's covered with the one below.

Reply:
✅ : https://github.com/algolia/recommend/pull/110/commits/9214aa08f43745d5dda1736e1fc1dd4118f5e038